### PR TITLE
Add an orders button. Add an active field to identify client existence.

### PR DESCRIPTION
**Pull request by MarrMika**

### CODE REVIEW

File: src/main/resources/templates/client/show_clients.ftl [hunk 1]
```diff
@@ -46,13 +43,16 @@
...
             <th scope="col">
-            <span style="margin-left: 27%;">
-            <form style='display:inline; width: 50px ' action="/clients/update/${client.id}" method="get">
-                <button class="btn btn-outline-success" type="submit">Edit</button>
-            </form>
-            <form style='display:inline; width: 50px' action="/clients/delete/${client.id}" method="post" >
-                <button class="btn btn-outline-danger" type="submit">Delete</button>
-            </form>
+            <span style="margin-left: 13%;">
+                <form style='display:inline; width: 50px' action="/orders/product/${client.id}" method="get">
```
Comment (Igor-Polatajko) [inline src/main/resources/templates/client/show_clients.ftl]: Do not use form to perform get request. Use regular `<a>` instead

File: src/main/java/com/pnu/ordermanagementapp/client/ClientDbAdapter.java [hunk 2]
```diff
@@ -19,7 +20,7 @@ public ClientDbAdapter(ClientRepository repository) {
 
     @Override
     public List<Client> findAll() {
-        return (List<Client>) repository.findAll();
+        return ((List<Client>)repository.findAll()).stream().filter(i->i.isActive()).collect(Collectors.toList());
```
Comment (Igor-Polatajko) [inline src/main/java/com/pnu/ordermanagementapp/client/ClientDbAdapter.java:23]: Do we need casting here?

Comment (Igor-Polatajko) [inline src/main/java/com/pnu/ordermanagementapp/client/ClientDbAdapter.java:23]: Formatting, every method on new line.
Also` i->i.isActive()` might be replaced with method refference

**Review by Igor-Polatajko — APPROVED** [review]
Looks good
Just a few comments

File: src/main/java/com/pnu/ordermanagementapp/client/ClientDbAdapter.java [hunk 3]
```diff
@@ -36,7 +37,8 @@ public void update(Client obj) {
     @Override
     public void delete(Long id) {
         Client client = findClientByIdOrThrowException(id);
-        repository.delete(client);
+        client.setActive(false);
```
Comment (Igor-Polatajko) [inline src/main/java/com/pnu/ordermanagementapp/client/ClientDbAdapter.java:40]: @volodymyrvynnyk Please, change logic in products part the same way.

File: src/main/resources/templates/client/show_clients.ftl (hunk 1 above)
Comment (MarrMika) [reply src/main/resources/templates/client/show_clients.ftl]: Okay, thanks

File: src/main/java/com/pnu/ordermanagementapp/client/ClientDbAdapter.java (hunk 2 above)
Comment (MarrMika) [reply src/main/java/com/pnu/ordermanagementapp/client/ClientDbAdapter.java:23]: Yeap, we need

Comment (MarrMika) [reply src/main/java/com/pnu/ordermanagementapp/client/ClientDbAdapter.java:23]: Ok

**Pull request merged**